gzip/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM squareslab/manybugs:base
 MAINTAINER Chris Timperley "christimperley@gmail.com"
 
 RUN sudo apt-get update && \
+	sudo apt-get upgrade -y subversion && \
```

Comment:
We might as well just add this to the big list of packages to install in the base Dockerfile.

Reply:
right on.